src/api/cow/index.ts

```diff
@@ -369,7 +369,7 @@ export class CowApi extends BaseApi {
     url: string,
     method: 'GET' | 'POST' | 'DELETE',
     baseUrl: string,
-    data?: unknown
+    data?: any
```

Comment:
Why changing the types here?

ESLint has suggestions to avoid `any`
![Screenshot 2023-01-11 at 14 00 45](https://user-images.githubusercontent.com/43217/211825340-011a8922-f60d-4f56-b470-61f15c98ea55.png)